### DISCUSSION TITLE
New UI

### DISCUSSION THREAD

**Discussion by PiixelNerd** [rafaelvcaetano/melonDS-android#1006] [Ideas]
This is my little idea for a new UI for melonDS!!
It also contains the idea of some new menus.
I'm designer, I dont have a lot of free time, but as I liked this port I decided to "help" in some way. If you like something about the idea, you can use it. As long as there is no monetization of the project.

![MelonDS new UI](https://user-images.githubusercontent.com/93597875/202850266-d53868c4-28a6-453a-867a-589d19748dc5.png)

**Comment by SombraShadow** [comment]
Thats cool bro,I hope Rafael reads this.

**Reply by PiixelNerd** [reply]
Thanks Bro! I'm glad you liked it! And I hope he sees it too!

**Reply by cyruzwings** [reply]
this is cool design concept , hope @rafaelvcaetano can implement some of this design

**Comment by rafaelvcaetano** [comment]
I read everything that's posted here.

Thanks a lot for the effort of making a suggestion with such a detailed explanation! Alternative layouts and the use of box covers is something that I have been considering for a long time but it's not a top priority. I have some ideas but I want to implement other features that could affect the final layout, so I don't want to have to change everything twice. I specially like the way you separated DSiWare ROMs from the rest. It's something I will keep in mind.

Regarding the redesigned top bar, what you suggested is not really viable. It contains a lot of options, which may not fit under the million screen sizes that the app must support. As such, I prefer to stick to the small list of options that are most important for the users in this specific screen.

**Reply by PiixelNerd** [reply]
Alright Rafael, no problem.  The idea was just to help, as I said, I'm a designer, I don't have much free time, but if you need something when your UI idea is ready, we can talk about it, if I have free time I can help you with the arts, icons, layouts and everything.
